v2/tools/generator/internal/astmodel/kubernetes_admissions_defaulter.go

```diff
@@ -50,14 +50,16 @@ func (d *DefaulterBuilder) AddDefault(f *resourceFunction) {
 // as well as helper functions that allow additional handcrafted defaults to be injected by
 // implementing the genruntime.Defaulter interface.
 func (d *DefaulterBuilder) ToInterfaceImplementation() *InterfaceImplementation {
-	lpr, ok := d.resourceName.PackageReference.AsLocalPackage()
+	group, version, ok := d.resourceName.PackageReference.GroupVersion()
 	if !ok {
-		panic(fmt.Sprintf("expected resource name %s to be a local package reference", d.resourceName.String()))
+		panic(fmt.Sprintf("expected resource name %s to not have an external package reference", d.resourceName))
```

Comment:
Wondering if we should rephrase this to:
`unexpected external package reference for resource name %s`?

Reply:
I like that phrasing. Done.